maintenance/preparation/idr0021-data-prep.md

```diff
@@ -0,0 +1,107 @@
+
+Workshop data preparation (idr0021)
+===================================
+
+This document details the steps to prepare data from IDR for a workshop demonstrating
+analysis with Fiji, usage of Map Annotations and OMERO.tables and filtering with OMERO.parade.
+We use IDR0021, which is a Project containing 10 Datasets with a total of ~400 Images.
+
+
+Download IDR data
+=================
+
+External users
+--------------
+
+Please contact the OME team for access to download IDR data.
```

Comment:
To be updated both internal and external can use
https://github.com/IDR/aspera-client-docker

```
$ docker run --rm -v /tmp:/data imagedata/download idr0021 . /data/ 

```